manager/controllers/advisory_detail.go

```diff
@@ -55,20 +46,11 @@ type AdvisoryDetailAttributes struct {
 	References       []string  `json:"references"`
 	RebootRequired   bool      `json:"reboot_required"`
 	ReleaseVersions  []string  `json:"release_versions"`
-}
-
-type AdvisoryDetailAttributesV1 struct {
-	AdvisoryDetailAttributes
-	Packages packagesV1 `json:"packages"`
-}
-
-type AdvisoryDetailAttributesV2 struct {
-	AdvisoryDetailAttributes
-	Packages packagesV2 `json:"packages"`
+	Packages         packages  `json:"packages"`
 }
 
 type packagesV1 map[string]string
```

Comment:
packagesV1 can be removed too, it was needed for V1 response which had different structure of packages response. V1 API used to return packages in a map[string]string format while since V2 the response is []string